package.json

```diff
@@ -87,7 +87,7 @@
     "dotenv": "^2.0.0",
     "extend": "~3.0.2",
     "file-type": "^7.7.1",
-    "ibm-cloud-sdk-core": "0.0.1",
+    "ibm-cloud-sdk-core": "0.0.3",
```

Comment:
Any reason why this is not a `^0.0.3`?

Reply:
I didn't do that on purpose but it actually is probably for the best since we don't want it to do a minor bump unintentionally. `v0.1.0` introduces breaking changes to the core.

Reply:
:+1